proposals/lip-0045.md

```diff
@@ -459,13 +457,13 @@ The chain validators substore holds information about the validators of other pa
 
 * The substore prefix is set to `SUBSTORE_PREFIX_CHAIN_VALIDATORS`.
 * Each store key is set to the ID of the partner chain `chainID`.
-* Each store value is the serialization of an object following the JSON schema `validatorsSchema` presented below.
-* Notation: For the rest of this proposal let `validators(chainID)` be the set of active validators for the chain identified by the store key `chainID`.
+* Each store value is the serialization of an object following the JSON schema `chainValidatorsSchema` presented below.
+* Notation: For the rest of this proposal let `validators(chainID)` be the entry in the chain validators substore identified by the store key `chainID`.
```

Comment:
We agreed to change the phrasing of this in all substores, see [this trello card](https://trello.com/c/K2WRLdXm/1423-fix-formulations-about-substore-notation) and [slack discussion](https://lightcurve.slack.com/archives/C011VEZTH7H/p1668773510318779?thread_ts=1668704574.601209&cid=C011VEZTH7H). 

So the new formulation should be: 

For the rest of this proposal let `validators(chainID)` be the value stored in the chain validators substore with store key `chainID`, deserialized using `chainValidatorsSchema`.

Reply:
updated them all